module-5.md

```diff
@@ -12,36 +12,102 @@ Planned:      1.25 hour
 
 ## 1: Topic introduction
 
-
+* Take up homework 
+* Discuss the need for keeping a pulse of the network
+  * What attributes are important to monitor
+  * How are these attributes measured?
+  * Identify units and what they mean
+  * Where have we seen it in our daily lives?
 
 ## 2: Hands-on and material exploration
 
-Monitor flow of network traffic using different tools as we do everyday digital things.
+Monitor flow of network traffic using different tools as we do every day digital things.
 
 ### Objectives
 
+Learn how these metrics affect us in our day to day lives, and in contrast how the same metrics affect others.
 
 ### Materials
 
+* Your cellphone
+* Some sort of public WiFi or some sort of internet connection
+* A Raspberry Pi 3 for each member labeled with the SSID for the node
+* SD cards with pre-flashed images with all required software
+* Laptops that can establish a SSH session to each Raspberry Pi (each running a Host AP with unique SSID)
+* 802.11s capable (e.g. TP-Link TL-WN722N v1) Radio for each Raspberry Pi 3
 
 ### Format
 
+Groups (or pairs) to start, class discussion then back to groups again
 
 ### Activity
 
-- Understanding the current internet:
-  - Speed test with library internet
-  - How much data does your app use
-
-- Grafana to show bandwidth usage:
-  - Locally stream music and video and see interface bandwidth
-  - What do your metrics mean?
-
-- Physical network topology (i.e. Toronto Mesh map) and Virtual peering topology (i.e. fc00.org)
+* Review your cell phone usage
+  * Identify your apps’ data usage
+  * Record top 3 apps that took up the most data
+  * Was this data cellular, or WiFi?
+* Public WiFi Speed Test
+  * Use a speed test tool to test the speed of the public WiFi, record the result.
+  * Is it faster or slower than the one you have at home? 
+  * Run the speed test again, record the result
+  
+* Discuss
+  * What unit is the data usage of your apps measured in?
+  * What does MB mean?
+  * What was your top app and how much MB did it use?
+  * Was that surprising?
+  * Would you be more/less surprised if it was cellular data? What if it was WiFi data? 
+    * Are different data sources worth more?
```

Comment:
Don't understand this.

Reply:
Example:
You do 3 gigs of Instagram data.
If it was 3 gigs of cell data, would it surprise you more then if it was 3 gigs of Wifi data.

The idea is that your CELL data is more precious since it costs more and you have a small amount of it. 
Most people will not have more then 3 gigs of cell data let alone 5 or 6, but have 300 gigs or more of WiFi data (home internet)

So eating up 3/3 gigs in Instagram will hurt allot more then 5/300.

Maybe needs to be reworded? Not quite sure how.